config/prod/prow/config_knative.yaml

```diff
@@ -464,12 +464,19 @@ presubmits:
   - unit-tests: false
   - integration-tests: false
   knative-sandbox/eventing-kafka:
+  - custom-test: integration-test-channel-consolidated
```

Comment:
Do you still need Docker-in-Docker to be enabled for these two new jobs? If yes, you'll also need to add `needs-dind: true` there.

Reply:
Yes; the script still needs that (if I'm not mistaken); I have added it back and regenerated the config script

Reply:
I don't think we need the docker-in-docker anymore eric - that was primarily for building the confluent/librdkafka from inside docker to avoid cross compiling with CGO, etc...